lib/models/change/middleware/notifications/change-request.js

```diff
@@ -0,0 +1,43 @@
+async function middleware (changes) {
+  const { wasNew } = changes;
+
+  let User = require('../../../company-user'),
+      Permission  = require('../../../permission'),
+      Notification = require('../../../notification');
+
+  if (!changes || !wasNew) {
+    return;
+  }
+
+  let permissionId = await Permission.findOne({ key: 'update changes' }).then(permission => {
```

Comment:
We are using async here, leverage it. We shouldn't need a `.then`.
```suggestion
  let permissionId = (await Permission.findOne({ key: 'update changes' }) || {})._id;
```

Reply:
🏁